plugins/oracle-bulk-loader/impl/src/test/java/org/pentaho/di/trans/steps/orabulkloader/OraBulkDataOutputTest.java

```diff
@@ -35,14 +35,9 @@
 
 import java.io.File;
 import java.io.IOException;
-
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Don't collapse imports with .*
We should explicitly import every class being used.